### PR TITLE
e2e: use ExitCleanly() in checkpoint tests

### PR DESCRIPTION
**Pull request by edsantiago**
Ongoing steps toward RUN-1907: replace Exit(0) with ExitCleanly()

Clean command-line replace, with one manual reversion (commented)

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: test/e2e/checkpoint_test.go [hunk 1]
```diff
@@ -1285,6 +1285,7 @@ var _ = Describe("Podman checkpoint", func() {
 
 		// As the container has been started with '--rm' it will be completely
 		// cleaned up after checkpointing.
+		// Cannot use ExitCleanly() because "skipping [ssh-agent-path] since it is a socket"
```
Comment (edsantiago) [inline test/e2e/checkpoint_test.go:1288]: This is the one case that failed on my laptop when I tried `ExitCleanly()`. I didn't take the time to figure out why.

**Comment by edsantiago** [comment]
Pushed with a skip-stderr-check-if-debian amendment.

**Comment by baude** [comment]
@cevich what is going on with the windows installer, agent is failing.  we need it working.

**Review by vrothberg — APPROVED** [review]
LGTM

**Review by giuseppe — APPROVED** [review]
LGTM

**Comment by cevich** [comment]
> @cevich what is going on with the windows installer, agent is failing. we need it working.

Just now getting to this.  @baude are we good now?  I chatted with Jason and I believe he fixed it, but I didn't watch his PR closely.

**Pull request merged**